[INFO 16:54:40] pymarl Running command 'my_main'
[INFO 16:54:40] pymarl Started run with ID "1"
[DEBUG 16:54:40] pymarl Starting Heartbeat
[DEBUG 16:54:40] my_main Started
[INFO 16:54:40] my_main Experiment Parameters:
[INFO 16:54:40] my_main 

{   'action_selector': 'epsilon_greedy',
    'add_value_last_step': True,
    'agent': 'rnn',
    'batch_size': 32,
    'batch_size_run': 32,
    'buffer_cpu_only': True,
    'buffer_size': 5000,
    'checkpoint_path': '',
    'common_reward': True,
    'env': 'gymma',
    'env_args': {   'key': 'WarehouseEnv-v0',
                    'pretrained_wrapper': None,
                    'seed': 937685613,
                    'time_limit': 200},
    'epsilon_anneal_time': 50000,
    'epsilon_finish': 0.05,
    'epsilon_start': 1.0,
    'evaluate': False,
    'gamma': 0.99,
    'grad_norm_clip': 10,
    'hidden_dim': 64,
    'hypergroup': None,
    'label': 'default_label',
    'learner': 'q_learner',
    'learner_log_interval': 10000,
    'load_step': 0,
    'local_results_path': 'results',
    'log_interval': 50000,
    'lr': 0.0005,
    'mac': 'basic_mac',
    'mixer': 'qmix',
    'name': 'qmix_warehouse',
    'obs_agent_id': True,
    'obs_last_action': True,
    'optim_alpha': 0.99,
    'optim_eps': 1e-05,
    'render': False,
    'repeat_id': 1,
    'reward_scalarisation': 'sum',
    'runner': 'parallel',
    'runner_log_interval': 10000,
    'save_model': False,
    'save_model_interval': 50000,
    'save_replay': False,
    'seed': 937685613,
    't_max': 200000,
    'target_update_interval_or_tau': 500,
    'test_greedy': True,
    'test_interval': 50000,
    'test_nepisode': 96,
    'use_cuda': True,
    'use_tensorboard': False,
    'use_wandb': False,
    'wandb_mode': 'offline',
    'wandb_project': None,
    'wandb_save_model': False,
    'wandb_team': None}

Process Process-1:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-2:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-3:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-4:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-5:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-6:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-7:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-8:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-9:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-10:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-11:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-12:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-13:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-14:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-15:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-16:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-17:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-18:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-19:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-20:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-21:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-22:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-23:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-24:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-25:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-26:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-27:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-28:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-29:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-30:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-31:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
Process Process-32:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/runners/parallel_runner.py", line 290, in env_worker
    env = env_fn.x()
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/__init__.py", line 33, in gymma_fn
    return GymmaWrapper(**kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/src/envs/gymma.py", line 39, in __init__
    self._env = gym.make(f"{key}", **kwargs)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 687, in make
    env_spec = _find_spec(id)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 531, in _find_spec
    _check_version_exists(ns, name, version)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 397, in _check_version_exists
    _check_name_exists(ns, name)
  File "/home/bhuv/Documents/bhuv-diffusion/epymarl/epymarl_env/lib/python3.8/site-packages/gymnasium/envs/registration.py", line 374, in _check_name_exists
    raise error.NameNotFound(
gymnasium.error.NameNotFound: Environment `WarehouseEnv` doesn't exist.
